rancher2/schema_cluster.go

```diff
@@ -407,7 +407,7 @@ func clusterFields() map[string]*schema.Schema {
 			Type:          schema.TypeList,
 			MaxItems:      1,
 			Optional:      true,
-			ConflictsWith: []string{"aks_config", "aks_config_v2", "eks_config", "eks_config_v2", "gke_config", "k3s_config", "oke_config", "rke_config", "rke2_config", "imported_config"},
```

Comment:
We should add the KEV1 operators (`aks_config`, `eks_config`, etc.) back to the `conflictsWith` list, otherwise users could define both KEV1 and KEV2 at the same time, which isn't supported afaik.